lib/model/query/submission-defs.js

```diff
@@ -103,7 +103,7 @@ module.exports = {
                 .orWhereIn('form_defs.keyId', keyIds))))
           .modify(applyODataFilter(options.filter))
           .where({ 'submissions.formId': formId, 'submissions.deletedAt': null })
-          .orderBy('submissions.id', 'desc')
+          .orderBy('submissions.createdAt', 'desc')
```

Comment:
This feels less urgent, but for consistency I think we should also update `Submission.getAllByFormId()` so that it orders by `createdAt` rather than `id`. That method is used by the regular/non-OData &hellip;/submissions endpoints.